src/main/kotlin/org/jitsi/jicofo/xmpp/XmppServices.kt

```diff
@@ -61,6 +77,11 @@ class XmppServices(xmppProviderFactory: XmppProviderFactory) {
         )
     } else null
 
+    private val avModerationHandler = AvModerationHandler(clientConnection.xmppConnection).also {
+        clientConnection.xmppConnection.addAsyncStanzaListener(it, MessageTypeFilter.NORMAL)
```

Comment:
I think we should use a sync listener and then offload to one of our IO threads.